notebook/04_portfolio_trade.ipynb

```diff
@@ -415,20 +373,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 9,
```

Comment:
"This toy strategy gets a Sharpe ratio 1.6 ". This is not true anymore as I reduced the number of stocks after the filtering.  We need to update this number

Reply:
Now it gives me 0.338. Thefore, does the following sentence applies (not sure if it is nice or note)?

> This toy strategy gets a Sharpe ratio 0.338 without considering the transaction cost. Nice!